Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Drinks/drinks_cans.yml

```diff
@@ -0,0 +1,18 @@
+- type: entity
+  parent: DrinkCanBaseFull
+  id: DrinkDrGibbBloodRedCan
+  name: Dr. Gibb Blood Red can
+  description:  A drink to quench YOUR bloodthirst.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 30
+        reagents:
+        - ReagentId: DrGibbBloodRed
+          Quantity: 30
+  - type: Drink
```

Comment:
this is inherited

Reply:
I just coppied it straight from DrGibb, so this is exactly how its done there